common/src/main/java/software/amazon/kms/common/KeyApiHelper.java

```diff
@@ -74,13 +73,6 @@ public ReplicateKeyResponse replicateKey(final ReplicateKeyRequest replicateKeyR
                 proxyClient.client()::replicateKey));
     }
 
-    public DescribeKeyResponse describeKey(final DescribeKeyRequest describeKeyRequest,
```

Comment:
Can you tell why is this being moved?

Reply:
+1, can you please tell why it is moved to AbstractKmsApiHelper.java

Reply:
KeyApiHelper is used only by key and replica-key. I needed describkey to be accessible from alias, so I moved to AbstractKmsApiHelper.